src/plugins/notebook/components/Notebook.vue

```diff
@@ -793,15 +793,29 @@ export default {
 
             return section.id;
         },
-        async newEntry(embed = null) {
+        async newEntry(embed, event) {
             this.startTransaction();
             this.resetSearch();
             const notebookStorage = this.createNotebookStorageObject();
             this.updateDefaultNotebook(notebookStorage);
             const id = await addNotebookEntry(this.openmct, this.domainObject, notebookStorage, embed);
+
+            const element = this.$refs.notebookEntries.querySelector(`#${id}`);
+            const entryAnnotations = this.notebookAnnotations[id] ?? {};
+            selectEntry({
+                element,
+                entryId: id,
+                domainObject: this.domainObject,
+                openmct: this.openmct,
+                notebookAnnotations: entryAnnotations
+            });
+            if (event) {
+                event.stopPropagation();
```

Comment:
how would we cover this line?

Reply:
I'm surprised this line isn't being covered as it was caught by tests! Ask @ozyx 😀